### PR TITLE
Restconversion

### PR DESCRIPTION
**Pull request by willoci**

### CODE REVIEW

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/model/Survey.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.objectcomputing.pulsesurvey.model;
+
+public class Survey {
```
Comment (mkimberlin) [inline send-surveys/src/main/java/com/objectcomputing/pulsesurvey/model/Survey.java]: So...this doesn't really describe a "Survey".  I expect that we will eventually need a "Survey" entity.  I would call this "SendSurveysCommand".

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java [hunk 2]
```diff
@@ -13,13 +14,18 @@
...
+import io.micronaut.http.annotation.Post;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@FunctionBean("send-surveys")
-public class SendSurveysFunction extends FunctionInitializer
-        implements Supplier<SendSurveys> {
-    private static final Logger LOG = LoggerFactory.getLogger(SendSurveysFunction.class);
+@Controller("/sendEmails")
+public class SendSurveysController {
```
Comment (mkimberlin) [inline send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java]: And let's just call this SurveysController...then make the root path "/surveys"

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java [hunk 3]
```diff
@@ -45,34 +51,43 @@ public void setGmailApi(GmailApi api) {
     /* call GetRandomEmails(what percentage of current employees) ->
     GetEmail(Google) -> SelectRandom -> GenerateKeys -> Map<String Email, String KeyUUID>
     Map gets returned */
-    @Override
-    public SendSurveys get() {
+
+    @Post(value = "/")
```
Comment (mkimberlin) [inline send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java]: Then change this to value="send" so the url will be /surveys/send

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/model/Survey.java (hunk 1 above)
Comment (willoci) [reply send-surveys/src/main/java/com/objectcomputing/pulsesurvey/model/Survey.java]: done

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java (hunk 2 above)
Comment (willoci) [reply send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java]: done

File: send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java (hunk 3 above)
Comment (willoci) [reply send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java]: done

**Pull request merged**